portal-web/test/functional/com/liferay/portalweb/paths/plugin/workflow/WorkflowMetricsSLA.path

```diff
@@ -0,0 +1,119 @@
+<html>
+<head>
+<title>WorkflowMetricsSLA</title>
+</head>
+<body>
+<table cellpadding="1" cellspacing="1" border="1">
+<thead>
+<tr><td rowspan="1" colspan="3">WorkflowMetricsSLA</td></tr>
+</thead>
+<tbody>
+<tr>
+	<td>ADD_NEW_SLA_BUTTON</td>
+	<td>//a[@class='btn btn-primary nav-btn nav-btn-monospaced']</td>
```

Comment:
hey Dennys, I think you could use a **contains** instead of **equal =** and with only one class that you believe is enough (e.g.: "btn-primary"), cause if one of those classes changes, this path will no longer work.